invenio_preservation_sync/services/service.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# Invenio-Preservation-Sync is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Service layer to process the Preservation Sync requests."""
+
+
+from flask import current_app, g
+from invenio_records_resources.services.uow import unit_of_work
+
+from invenio_preservation_sync.api import PreservationInfoAPI
+from invenio_preservation_sync.errors import PermissionDeniedError
+from invenio_preservation_sync.services.permissions import (
+    DefaultPreservationInfoPermissionPolicy,
+)
+
+
+class PreservationInfoResult(object):
```

Comment:
Move to results.py, add schema dump like: https://github.com/inveniosoftware/invenio-banners/blob/master/invenio_banners/services/results.py